pkg/generation/cnr.go

```diff
@@ -97,7 +97,7 @@ func GenerateCNR(nodeGroup atlassianv1.NodeGroup, nodes []string, name, namespac
 		}
 	}
 
-	return atlassianv1.CycleNodeRequest{
+    return atlassianv1.CycleNodeRequest{
```

Comment:
nit: due to what I assume is IDE settings we're currently mixing tabs with spaces. Please ensure your IDE is configured for tabs or fix the linter to pick this up and fix it, so we don't get a mix in the files.